templates/layouts/bootstrap.html.ep

```diff
@@ -128,10 +129,14 @@
               </div><!-- /.navbar-collapse -->
           </div>
       </nav>
+    </div>
       <div class="container" id="content">
           %= content
       </div>
 
+    % my $fullscreen = $self->param('fullscreen');
```

Comment:
You need to stash this variable in the controller (Main.pm), so you don't have to define it here again.